src/components/EstablishmentCard.tsx

```
@@ -0,0 +1,67 @@
+    >
+      <Image
+        src={establishment.logo}
+        width={52}
+        height={52}
+        style={{
+          borderRadius: '50%',
+          objectFit: 'cover',
+          objectPosition: 'center',
+        }}
+        alt={establishment.name}
+      ></Image>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)